BTSPocket/Controllers/Profile/ProfileViewController.swift

```diff
@@ -7,23 +7,125 @@
 
 import UIKit
 
-class ProfileViewController: UIViewController {
+private enum ProfileSections: String, CaseIterable {
+    case basicInfo = "INFO"
+    case skills = "SKILLS"
+    case experience = "EXPERIENCE"
+}
 
+class ProfileViewController: UIViewController {
+    
+    // MARK:- Outlets
+    @IBOutlet weak private var tableView: UITableView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        // register table view cells for the table view
+        self.tableView.register(UINib(nibName: "ProfileTableViewCell", bundle: nil), forCellReuseIdentifier: "ProfileTableViewCell")
+        self.tableView.register(UITableViewCell.self, forCellReuseIdentifier: "DefaultCell")
+        self.tableView.separatorStyle = UITableViewCell.SeparatorStyle.none
+    }
+}
 
-        // Do any additional setup after loading the view.
+// MARK:- Table view data source
+extension ProfileViewController: UITableViewDataSource, UITableViewDelegate {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        if BTSApi.shared.profileSession?.skills?.count ?? 0 > 0 {
+            if BTSApi.shared.profileSession?.experiences?.count ?? 0 > 0 {
+                return 3
+            } else {
+                return 2
+            }
+        } else if BTSApi.shared.profileSession?.experiences?.count ?? 0 > 0 {
+            if BTSApi.shared.profileSession?.skills?.count ?? 0 > 0 {
+                return 3
+            } else {
+                return 2
+            }
+        } else {
+            return 1
+        }
     }
     
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch ProfileSections.allCases[section].rawValue {
+        case "INFO":
+            return ""
+        case "SKILLS":
+            return "Skills"
+        case "EXPERIENCE":
+            return "Experience"
+        default:
+            return ""
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch ProfileSections.allCases[section].rawValue {
+        case "INFO":
+            return 1
+        case "SKILLS":
+            return BTSApi.shared.profileSession?.skills?.count ?? 0
+        case "EXPERIENCE":
+            return BTSApi.shared.profileSession?.experiences?.count ?? 0
+        default:
+            return 1
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
+        switch ProfileSections.allCases[indexPath.section].rawValue {
```

Comment:
Same case than previous, String Enum is not the best way to implement due to future localization